src/dreyfus_httpd.erl

```diff
@@ -35,6 +35,15 @@ handle_search_req(#httpd{method=Method, path_parts=[_, _, _, _, IndexName]}=Req
         include_docs = IncludeDocs,
         grouping = Grouping
     } = parse_index_params(Req),
+    QueryArgs1 = QueryArgs#index_query_args{
+        partitioned = mem3:is_partitioned(hd(mem3:shards(DbName))) % hideous
```

Comment:
if there's a dependency PR for is_partitioned, perhaps we reference it in this PR?